### ISSUE TITLE
whether the code is complete

### ISSUE THREAD

**Issue by Guanerhua** [Dingyuan-Zheng/ctf-UDA#1]
where is the code you modify ，tell me ,please

**Comment by Dingyuan-Zheng** [comment]
> where is the code you modify ，tell me ,please

Yes, the code is complete. The code implementation for our proposed method can be mainly found in the python files, 'mmt_train_kmeans.py', 'trainers.py' 'dukemtc.py/market.py/msmt17.py', and we also modified some relevant python files. 

You can prepare datasets and train/test the model following the instructions of MMT, since our code framework is based on MMT. 

Besides, we also provide some training/test log files under the folder 'ctf-UDA/logs/'. The test results in the attached log files might be slightly higher than the results we reported in our paper. You can compare your results with those reported in our paper (lower one).

**Comment by Guanerhua** [comment]
Thanks,I find what you modify.But in train.py line 298  :             
 for targets_idx in range(8):
       sim_distribution = []
       for idx in range(cluster_center.shape[0]):
                    sim = torch.dist(cluster_center[targets[targets_idx*4]], cluster_center[idx], p=2)
I mean that it is Eucidean distance between the abchor centroid and the other cluster centroids. but i cant understand the "cluster_center[targets[targets_idx*4]]".
